package.json

```diff
@@ -131,6 +131,26 @@
                 "com.apple.security.device.camera": true
             }
         },
+        "dmg": {
+            "background": "assets/background.tiff",
```

Comment:
is this included at build time? if yes it should be in the build folder, not assets

Reply:
The dmg iirc is created after vesktop is built

Reply:
try moving it into build and see if it still works

Reply:
Done, and yes it compiles fine.

Reply:
yep i figured! assets/ is for runtime assets, build/ for compile time only stuff

Reply:
I didn't even know that tbf, I just thought it would be appropriate to have it be in another directory